addons/csw/functions/fnc_staticWeaponInit.sqf

```diff
@@ -63,12 +63,13 @@ if (hasInterface && {!(_typeOf in GVAR(initializedStaticTypes))}) then {
         [_typeOf, 0, ["ACE_MainActions"], _disassembleAction] call EFUNC(interact_menu,addActionToClass);
     };
 
+    if (GVAR(ammoHandling) == 0) exitWith {};
```

Comment:
this is not the same

Reply:
Yeah, my bad. Misread `||` instead of `&&`.